requirements/requirements_ansible.txt

```diff
@@ -97,20 +99,20 @@ pyparsing==2.4.0          # via packaging
 python-dateutil==2.6.1    # via adal, azure-storage, botocore
 pyvmomi==6.5
 pywinrm[kerberos]==0.3.0
-pyyaml==5.1               # via azure-cli-core, knack, openstacksdk
+pyyaml==5.1               # via ansible-tower-cli, azure-cli-core, knack, openstacksdk
 requests-credssp==1.0.2
 requests-kerberos==0.12.0  # via pywinrm
 requests-ntlm==1.1.0      # via pywinrm
 requests-oauthlib==1.2.0  # via msrest
-requests==2.21.0
+requests==2.22.0
```

Comment:
not sure if this is safe to upgrade

Reply:
I _think_ this is okay.  If you're referring to the behavioral change here:

<img width="695" alt="image" src="https://user-images.githubusercontent.com/214912/60908247-6443fb80-a249-11e9-9c30-b067c364f921.png">

This is actually a change in `urllib>=1.25`, not `requests` itself:
https://urllib3.readthedocs.io/en/latest/user-guide.html#ssl